pkgs/default.nix

```diff
@@ -5,15 +5,29 @@
 
 let
   inherit (import <microgram/sdk.nix>) pkgs lib;
+  inherit (import <microgram/lib>) mapcats;
   inherit (pkgs)
-    pythonPackages perlPackages haskellPackages
-    stdenv_32bit gnome
-    stdenv fetchurl newScope;
+    pythonPackages perlPackages haskellPackages stdenv_32bit gnome stdenv
+    fetchurl newScope;
   inherit (lib)
-    concatMapStringsSep overrideDerivation optionalAttrs makeSearchPath;
+    concatMapStringsSep overrideDerivation optionalAttrs makeSearchPath
+    concatStringsSep;
 
   fns = rec {
 
+
+    exportSessionVariables = mapcats (k: v: "export ${k}=${v}") {
```

Comment:
okay I'll admit I'm confused by `fns` (functions?) that aren't meant to be pure
but it makes sense to move this one to lib, now that it exists